test/integration.js

```diff
@@ -16,16 +19,21 @@ test('integration', async () => {
 
 test('retrieval check for our CID', async () => {
   const spark = new Spark()
-  spark.getRetrieval = async () => ({ cid: KNOWN_CID })
+  spark.getRetrieval = async () => ({ cid: KNOWN_CID, minerId: OUR_FAKE_MINER_ID })
+  spark.lookupMinerPeerId = async (minerId) => {
+    assertEquals(minerId, OUR_FAKE_MINER_ID)
+    return FRISBEE_PEER_ID
+  }
   const measurementId = await spark.nextRetrieval()
   const res = await fetch(`https://api.filspark.com/measurements/${measurementId}`)
   assert(res.ok)
   const m = await res.json()
   const assertProp = (prop, expectedValue) => assertEquals(m[prop], expectedValue, prop)
 
   assertProp('cid', KNOWN_CID)
-  // TODO - spark-api does not record this field yet
-  // assertProp('indexerResult', 'OK')
```

Comment:
Unrelated change; I am happy to do it in a different pull request if you prefer.

Reply:
On the second thought, let's do that. See https://github.com/filecoin-station/spark/pull/57